templates/citizen_action_manager/_layout.html.twig

```diff
@@ -54,6 +54,7 @@
                     {{ form_errors(citizen_action_form.finishAt) }}
                     {{ form_label(citizen_action_form.finishAt, 'Fin', { label_attr: { class: 'text--bold' } }) }}
                     {{ form_widget(citizen_action_form.finishAt, { attr: { class: 'citizen_action__finish_at l__row' } }) }}
+                    <div class="form__info">Votre action ne peut pas durer plus de 3 jours.</div>
```

Comment:
To be consistent with the wording in the project: `action` -> `action citoyenne`.

Reply:
done 👍 tnx